### PR TITLE
Add version history for 5.6.0

### PR DESCRIPTION
**Pull request by dgault**
See https://github.com/openmicroscopy/bioformats/milestone/55?closed=1
for list of included PRs.

### CODE REVIEW

File: docs/sphinx/about/whats-new.rst [hunk 1]
```diff
@@ -1,6 +1,68 @@
...
+  and consumed as 'org.openmicroscopy:ome-codecs' artifact from Maven Central
+* removed components/forks/jai - decoupled to the new
+  `ome/ome-jai GitHub repository <https://github.com/ome/ome-jai>`_
+  and consumed as part of 'org.openmicroscopy:ome-codecs' artifact from Maven Central
+* replaced components/formats-api/codecs classes with wrappers
+* replaced components/formats-bsd/codecs classes with wrappers
+
+Updated build system:
+
+* ant now removes bundles build files during 'clean' to prevent a mix of dependencies
```
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.rst]: bundled

File: docs/sphinx/about/whats-new.rst [hunk 2]
```diff
@@ -1,6 +1,68 @@
...
+   - improved performance when using options to concatenate multiple series together
+* TiffSaver
+   - made performance improvements to prevent the writing of a new IFD for each tile, 
+     resulting in significant file size reductions for images with a large quantity of tiles
+
+Documentation improvements:
+
+* updated website and URL links for new ``openmicroscopy.org`` website
+* added missing Andor SIF to supported formats page
+* added a new page ``Working with whole slide images`` outlining the API support
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: Maybe worth using `:doc:` markup

File: docs/sphinx/about/whats-new.rst [hunk 3]
```diff
@@ -1,6 +1,68 @@
...
+     resulting in significant file size reductions for images with a large quantity of tiles
+
+Documentation improvements:
+
+* updated website and URL links for new ``openmicroscopy.org`` website
+* added missing Andor SIF to supported formats page
+* added a new page ``Working with whole slide images`` outlining the API support 
+  for pyramids/resolutions
+* fixed broken documentation links for external resources which are no longer available
+* updated the style of sphinx documentation
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: sphinx -> Sphinx

File: docs/sphinx/about/whats-new.rst [hunk 4]
```diff
@@ -1,6 +1,68 @@
...
+
+Component architecture changes/decoupling:
+
+* decoupled image encoding and decoding routines to the new
+  `ome/ome-codecs GitHub repository <https://github.com/ome/ome-codecs>`_
+  and consumed as 'org.openmicroscopy:ome-codecs' artifact from Maven Central
+* removed components/forks/jai - decoupled to the new
+  `ome/ome-jai GitHub repository <https://github.com/ome/ome-jai>`_
+  and consumed as part of 'org.openmicroscopy:ome-codecs' artifact from Maven Central
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: `org.openmicroscopy:ome-jai`

File: docs/sphinx/about/whats-new.rst [hunk 5]
```diff
@@ -1,6 +1,68 @@
...
+* replaced components/formats-api/codecs classes with wrappers
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: is there more to be said about the wrapper classes here @rleigh-codelibre ?

File: docs/sphinx/about/whats-new.rst (hunk 1 above)
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: or maybe `Ant now removes the build files of the bundles...`

File: docs/sphinx/about/whats-new.rst (hunk 5 above)
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.rst]: Maybe "wrappers around ome-codecs" for clarity, but not much else to add here.

File: docs/sphinx/about/whats-new.rst [hunk 6]
```diff
@@ -41,12 +41,12 @@ Bug fixes and improvements:
 
 Documentation improvements:
 
-* updated website and URL links for new ``openmicroscopy.org`` website
-* added missing Andor SIF to supported formats page
-* added a new page ``Working with whole slide images`` outlining the API support 
+* updated website and URL links for new <http://www.openmicroscopy.org>`_ website
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: Maybe `OME Website <https://www.openmicroscopy.org>`_ as the brackets are currently rendered.

**Pull request merged**